src/Accounts/AccountsForm.js

```diff
@@ -63,82 +93,110 @@ class AccountsForm extends Component {
                 />
               </Col>
               <Col xs={12}>
-                <Field label={<FormattedMessage id="ui-organizations.accounts.description" />} name={`${elem}.description`} id={`${elem}.description`} component={TextField} fullWidth />
+                <Field
+                  component={TextField}
+                  fullWidth
+                  label={<FormattedMessage id="ui-organizations.accounts.description" />}
+                  name={`${elem}.description`}
+                />
               </Col>
               <Col xs={12}>
-                <Field label={<FormattedMessage id="ui-organizations.accounts.payable" />} name={`${elem}.appSystemNo`} id={`${elem}.appSystemNo`} component={TextField} fullWidth />
+                <Field
+                  component={TextField}
+                  fullWidth
+                  label={<FormattedMessage id="ui-organizations.accounts.payable" />}
+                  name={`${elem}.appSystemNo`}
+                />
               </Col>
               <Col xs={12}>
                 <Field
+                  component={Select}
+                  dataOptions={paymentMethodDD}
+                  fullWidth
                   label={<FormattedMessage id="ui-organizations.accounts.paymentMethod" />}
                   name={`${elem}.paymentMethod`}
                   placeholder=" "
-                  dataOptions={paymentMethodDD}
-                  validate={[Required]}
-                  component={Select}
-                  fullWidth
                   required
+                  validate={[Required]}
                 />
               </Col>
             </Row>
           </Col>
-          <Col xs={12} md={6}>
+          <Col
+            xs={12}
+            md={6}
+          >
             <Row>
               <Col xs={12}>
                 <Field
+                  component={Select}
+                  fullWidth
                   label={<FormattedMessage id="ui-organizations.accounts.account.accountStatus" />}
```

Comment:
Are you sure that account status it's the same as organization status?

Reply:
No, but it has the same statuses.